### ISSUE TITLE
[ISSUE] - Sidebar UI Issue when we have content on right side means the middle

### ISSUE THREAD

**Issue by thecuriousteam** [thecuriousteam/PyLibLog#104]
Please find the screenshot attached below

This issue is observed with the following pull request  #97 

![image](https://github.com/thecuriousteam/PyLibLog/assets/120193300/1f39bb52-2a3a-4b61-9d96-da8bbf8002c9)

**Comment by shrey802** [comment]
@thecuriousteam solving this

**Comment by thecuriousteam** [comment]
Yes @shrey802 you can work on it, please sync the fork and take the latest pull request

**Comment by Tooba-Amjad** [comment]
@thecuriousteam wanna work on it

**Comment by nishantshukla2884** [comment]
hey @thecuriousteam i want contribute to this issue , can you please elaboate
